lib/galaxy/webapps/galaxy/api/job_files.py

```diff
@@ -114,6 +123,47 @@ def create(self, trans, job_id, payload, **kwargs):
                 pass
         return {"message": "ok"}
 
+    @expose_api_raw_anonymous_and_sessionless
```

Comment:
I think Mira is right here, we probably need to return valid JSON for TUS 2.0, so `@expose_api_anonymous_and_sessionless` is the right decorator here and for `tus_hooks`